pkg/monitor/cluster/podconditions_test.go

```diff
@@ -141,3 +141,43 @@ func TestEmitPodContainerStatuses(t *testing.T) {
 	ps, _ := cli.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{})
 	mon._emitPodContainerStatuses(ps)
 }
+
+func TestEmitPodRestartCounts(t *testing.T) {
```

Comment:
It is usually `Test` + `NameOfTheFunctionYouTest`. For example:

* `_emitPodConditions` -> `TestEmitPodConditions`
* `_emitPodContainerStatuses` -> `TestEmitPodContainerStatuses`

```suggestion
func TestEmitPodContainerRestartCounter(t *testing.T) {
```